Bringing attention to the use of the Normalize function which is being deprecated

Output in the CoreDNS container:

```text
authoritative-dns-coredns-7c46c46c57-2ccss [INFO] Reloading                                                                                                                                                                  │
│ authoritative-dns-coredns-7c46c46c57-2ccss [WARNING] An external plugin (/go/pkg/mod/github.com/networkservicemesh/fanout@v1.9.1/setup.go line 107) is using the deprecated function Normalize. This will be removed in a fu │
│ ture versions of CoreDNS. The plugin should be updated to use OriginsFromArgsOrServerBlock or NormalizeExact instead.                                                                                                        │
│ authoritative-dns-coredns-7c46c46c57-2ccss [WARNING] An external plugin (/go/pkg/mod/github.com/networkservicemesh/fanout@v1.9.1/setup.go line 107) is using the deprecated function Normalize. This will be removed in a fu │
│ ture versions of CoreDNS. The plugin should be updated to use OriginsFromArgsOrServerBlock or NormalizeExact instead.                                                                                                        │
│ authoritative-dns-coredns-7c46c46c57-2ccss [INFO] 127.0.0.1:51879 - 10951 "HINFO IN 363186276156127583.2967028074172956578. udp 56 false 512" - - 0 0.000021983s                                                             │
│ authoritative-dns-coredns-7c46c46c57-2ccss [ERROR] plugin/errors: 2 363186276156127583.2967028074172956578. HINFO: plugin/loop: no next plugin found                                                                         │
│ authoritative-dns-coredns-7c46c46c57-2ccss [INFO] plugin/reload: Running configuration SHA512 = 3253fe25877b0cd47445bfe42a542c1dd342d05fdb2689a69214b7766c8053147e0807539dbb5ff5cdfb35a65863cccb5054daeb02d7f56c5bc2d6721938 │
│ d9a2
```

Is this already in the plans for being looked at?

Thank you

--- a/setup.go
+++ b/setup.go
@@ -101,10 +101,10 @@ func parseFanout(c *caddy.Controller) (*Fanout, error) {
 
 func parsefanoutStanza(c *caddyfile.Dispenser) (*Fanout, error) {
 	f := New()
-	if !c.Args(&f.from) {
+	if !c.NextArg() {
 		return f, c.ArgErr()
 	}
-	f.from = plugin.Host(f.from).Normalize()
+	f.from = plugin.Host(c.Val()).NormalizeExact()[0]
 	to := c.RemainingArgs()
 	if len(to) == 0 {
 		return f, c.ArgErr()

